src/decisionengine/framework/modules/logging_configDict.py

```diff
@@ -3,6 +3,9 @@
 """
 import structlog
 
+LOGGERNAME = "decisionengine"
+CHANNELLOGGERNAME = "channelog"
```

Comment:
"channel" (instead of channelog) would be more consistent w/ "decisionengine"

Reply:
"channel" is confusing in the logs because we have a dict item called "channel" that corresponds to the name of the channel that is producing the log.  See here:
{"module": "TaskManager", "channel": "job_classification", "event": "transform put data", "logger": "channelog", "level": "info", "timestamp": "2021-08-31 15:06:16"}

I would rather change the "decisionengine" name to "decisionenginelog" than "channelog" to "channel".

Reply:
I think that one is a key, one is a value, they should be sufficiently distinct, and personally, I prefer the name not to include what the object is ("channel" log and not "channellog" log, similar to "Benji" dog and not "Benjidog" dog). But any name is correct, so it is OK